packages/ui/src/common/components/typography/Fraction.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+
+import { Colors } from '../../constants'
+
+import { TextInlineMedium } from './Text'
+
+interface FractionProps {
+  num: string | number
+  den: string | number
```

Comment:
Those props are too cryptic. Having full names in props helps to understand what they are for.

Here, it should be rather: `numerator`  & `denominator` as you've named components.